docs/oracle-operator-manual.md

```diff
@@ -38,15 +38,15 @@ In order to launch oracle daemon on your machine, you need to have several thing
 
 ## The oracle daemon
 
-The oracle daemon is a simple Python app that watches the Beacon chain and pushes the data to the LidoOracle Smart Contract: [Mainnet](https://etherscan.io/address/0x442af784A788A5bd6F42A01Ebe9F287a871243fb) / [Görli](https://goerli.etherscan.io/address/0x1643E812aE58766192Cf7D2Cf9567dF2C37e9B7F).
+The oracle daemon is a simple Python app that watches the Beacon chain and pushes the data to the LegacyOracle Smart Contract: [Mainnet](https://etherscan.io/address/0x442af784A788A5bd6F42A01Ebe9F287a871243fb) / [Görli](https://goerli.etherscan.io/address/0x1643E812aE58766192Cf7D2Cf9567dF2C37e9B7F).
```

Comment:
this is not correct

Reply:
the whole document is not correct as I understand